roachtest: deflake splits/load/uniform/nodes=3

tbg opened the pull request:
The workload that was supposed to trigger the splits ran only for ~35s,
which exposed the test to significant amounts of randomness. Ditch the
overly clever computation and run for 10 minutes, which is well above
(by magnitudes) what's required for the splits to take place.

Closes #34359.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/37875)
<!-- Reviewable:end -->

tbg committed:
roachtest: deflake splits/load/uniform/nodes=3

The workload that was supposed to trigger the splits ran only for ~35s,
which exposed the test to significant amounts of randomness. Ditch the
overly clever computation and run for 10 minutes, which is well above
(by magnitudes) what's required for the splits to take place.

Closes #34359.

Release note: None

petermattis reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/37875#-:-LfuUbuc1uHZaHMMryil:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @petermattis)



<!-- Sent from Reviewable.io -->

tbg commented:
bors r=petermattis
